lib/opam_build.mli

```diff
@@ -1,7 +1,10 @@
-(** Generate a Dockerfile for building all the opam packages in the build context. *)
-val dockerfile :
-  base:string ->
-  info:Analyse.Analysis.t ->
-  repo:Current_github.Repo_id.t ->
+(** Build and test all the opam packages.
+    [~variant] is the variant of the "ocurrent/opam" image.
+    [~repo] is used to identify a build cache for duniverse projects. *)
```

Comment:
That's a weird thing to expose in documentation; if I wanted to see what it's used for, I'd read the code. Shouldn't it tell me what `repo` actually _is_?

Reply:
Sorry, I couldn't find a use for this argument that fits. What would you think of resolving this earlier and taking an optional list of caches here instead ?

Reply:
Resolving this earlier means not being able to test it :slightly_smiling_face: Why not just something simple like `[~repo] is the ID of the repository-under-test on GitHub`. 

You could even make it optional and have it default to `{  Github.Repo_id.owner = "local"; name = "test"  }`, which is what's being done in `Pipeline.local_test`.

Reply:
I updated the doc :)
I don't think we should give it a default value. This value will be used depending on a flag in `analysis`.